Comments/ClientApp/src/styles/forms.scss

```diff
@@ -165,3 +165,106 @@ $colour-border-focus: $colour-nice-blue-3;
   }
 
 }
+
+
+// temporary manual addition of checkbox styling from the design system
```

Comment:
let's hope so

Reply:
at the moment all the form stuff is manually added from the design system so it was probably a misleading comment!